tools/ci_build/build.py

```diff
@@ -2162,6 +2162,27 @@ def gen_ort_aten_ops(eager_root_dir):
             eager_root_dir = os.path.join(source_dir, "orttraining", "orttraining", "eager")
             gen_ort_aten_ops(eager_root_dir)
 
+        # Note: Please make sure submodule onnxruntime-extensions is already pulled/synced now.
+        # Steps: 1. generate _selectedoplist.cmake by operators config file
+        #        2. set cmake option to select operators from onnxruntime-extensions
+        if is_reduced_ops_build(args) and args.enable_onnxruntime_extensions:
```

Comment:
Also, if the NO_EXCEPTION enabled, also update the configuration for the OCOS too.

Reply:
Thanks for comment. Have added `OCOS_ENABLE_CPP_EXCEPTIONS` option in CMakeLists.txt file.